### PR TITLE
Bring beta closer to mockups

### PR DESCRIPTION
**Pull request by fureigh**
Addresses issues #160 and #162.

This currently includes commits from other pending PRs (true confessions); I can finish cherry-picking in the morning but figured it was best to share the current state.

### Outstanding issues
 * Feed sources (i.e., Tumblr RSS feeds) aren't exported as part of standard configuration. We'll need to account for them separately.
 * I intend to link the social media presence names in the upper right corner to corresponding anchors throughout the page. I'll save that for a separate PR, but want to be clear that at the moment they're just placeholders.

### CODE REVIEW

**Comment by fureigh** [comment]
Here's how it currently looks on my local machine:

screen shot 2018-03-22 at 1 23 33 am

File: web/sites/default/config/views.view.nsf_blog.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ display:
         type: none
         options: {  }
       cache:
-        type: tag
+        type: none
```
Comment (cmc333333) [inline web/sites/default/config/views.view.nsf_blog.yml]: Is the time-based caching mentioned in this commit message worth making an issue over?

Comment (fureigh) [reply web/sites/default/config/views.view.nsf_blog.yml]: I'll make a quick catchall 'opportunities for performance improvements' ticket.

**Comment by fureigh** [comment]
Thanks, @cmc333333!

**Pull request merged**